include/cantera/oneD/StFlow.h

```diff
@@ -205,6 +205,21 @@ class StFlow : public Domain1D
 
     void solveEnergyEqn(size_t j=npos);
 
+    //! Get the solving stage
+    virtual size_t getSolvingStage() const;
+
+    //! Set the solving stage
+    virtual void setSolvingStage(const size_t stage);
```

Comment:
Some extra description to highlight that these are used by `IonFlow` (like in the Python docstrings) would be useful.